app/vep/vep_resources.py

```diff
@@ -54,12 +61,30 @@ async def submit_vep(request: Request):
         else:
             raise Exception
     except MaxBodySizeException:
-        return HTTPException(
-            status_code=status.HTTP_413_REQUEST_ENTITY_TOO_LARGE,
-            detail="Maximum file size limit exceeded.",
-        )
+        return response_error_handler(result={"status": 413})
     except Exception as e:
-        return HTTPException(
-            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
-            detail="Could not submit VEP job/workflow.",
-        )
+        return response_error_handler(result={"status": 500})
+
+
+@router.get("/submissions/{job_id}/status", name="submission_status")
+async def submit_vep(request: Request, job_id: str):
+    try:
+        workflow_status = await get_workflow_status(job_id)
+        return JSONResponse(content=workflow_status.dict())
+
+    except HTTPError as http_error:
+        if http_error.response.status_code == 403:
+            response_msg = json.dumps(
+                {
+                    "status_code": status.HTTP_404_NOT_FOUND,
+                    "details": f"A submission with id {job_id} was not found",
+                }
```

Comment:
This isn't what I wrote in the spec 🤷‍♂️ Do we want to change the spec?

Reply:
In the spec for this specific error  we're drifting from the general error format that we use, I'm ready to either updating the spec or code but I thought you should know that before.

Reply:
I did not know about the general error format. Personally, I think the status field in the body is excessive; but if that's our general error format... Let's see what others think.

Reply:
As far as I remember resolver, refget proxy and metadata API are using the same error_response file that generates the same format I thought you're aware of it!  Let's go ahead with using the general format for now then since others are already using it.

Reply:
spec is updated.

Reply:
Here we are reimplementing `http_404_not_found` function from  `response_error_handler` to change the value of the details field. Would it make sense to instead update `response_error_handler` to accept a custom status message?